### DISCUSSION TITLE
Composition API Switch Button in the docs

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [vuejs/pinia#2502] [Ideas]
Hi, 

it would be nice (and consistent) to have in the docs a switch button to switch between Options/Composition API store syntaxes.
Label can be `Options / Setup Function`

![image](https://github.com/vuejs/pinia/assets/10191085/eb1da3a2-fc52-40ff-90e6-64c3cff4b201)

**Comment by posva** [comment]
Duplicate of #2426

**Reply by angelhdzmultimedia** [reply]
Wow... I'm having serious memory loss. Weird. Sorry for the inconveniences caused.
